include/hip/devicelib/type_casting_intrinsics.hh

```diff
@@ -98,4 +96,9 @@ EXPORT float __ull2float_rn(unsigned long long int x);
 EXPORT float __ull2float_ru(unsigned long long int x);
 EXPORT float __ull2float_rz(unsigned long long int x);
 
+extern "C" {
+__device__ long long int __double_as_longlong(double x);
+__device__ double __longlong_as_double(long long int x);
+}
+
```

Comment:
We should just modify the `EXPORT` macro if it's not correct.

Reply:
I don’t dare to adjust the `EXPORT` macro just for these two functions. I don’t know which of the other device functions rely on the current macro definition and whether they are well tested for regressions coming from the change. Doesn’t seem like a worthwhile effort.